src/gcp/auth.ts

```diff
@@ -1,6 +1,56 @@
 import { Client } from "../apiv2";
 import { identityOrigin } from "../api";
 
+interface MfaEnrollment {
+  mfaEnrollmentId: string;
+  displayName: string;
+  enrolledAt: string;
+  phoneInfo?: string;
+  emailInfo?: {
+    emailAddress: string;
+  };
+  unobfuscatedPhoneInfo?: string;
+}
+
+interface UserInfo {
+  localId: string;
+  email: string;
+  displayName: string;
+  language: string;
+  photoUrl: string;
+  timeZone: string;
+  dateOfBirth: string;
+  passwordHash: string;
+  salt: string;
+  version: number;
+  emailVerified: boolean;
+  passwordUpdatedAt: number;
+  providerUserInfo: {
+    providerId: string;
+    displayName: string;
+    photoUrl: string;
+    federatedId: string;
+    email: string;
+    rawId: string;
+    screenName: string;
+    phoneNumber: string;
+  }[];
+  validSince: string;
+  disabled: boolean;
+  lastLoginAt: string;
+  createdAt: string;
+  screenName: string;
+  customAuth: boolean;
+  rawPassword?: string;
```

Comment:
Probably safe to omit input only fields.

Reply:
Good call, removed.